src/const/const.js

```diff
@@ -0,0 +1 @@
+export const LIMIT = 8;
```

Comment:
Let's add more details to this by moving this file to the relevant component. Rename the file and rename the constant.